Test/Integration/_files/transaction/models/quote.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
@ryanreeves-taxjar Are we still using this file to generate a quote object?

Reply:
@jakedjohnson we are not.  I left it in for future use, but it could be removed from this commit without error.